ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -0,0 +1,98 @@
+@{
+    var notificationId = ViewData["NotificationId"];
+}
+
+<div class="notification-overview-type-and-edit-container">
+    <h3 class="notification-details-type"> Clinical details </h3>
+    <a href="./Edit/ClinicalDetails?id=@notificationId" class="notification-edit-link"> Edit </a>
+</div> 
+<div class="notification-overview-details-container">
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Sites of Disease </div>
+            <div class="empty-cell-min-height"> @Model.SitesOfDisease </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Drug resistance profile </div>
+            <div>  </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Sample details </div>
+            <div class="empty-cell-min-height"> @(@Model.Notification.ClinicalDetails.NoSampleTaken ? "No" : "Yes") </div>
+        </div>
+    </div>
+    <div class="nhsuk-grid-row notification-overview-details">
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Symptom onset date </div>
+            <div class="empty-cell-min-height"> @Model.Notification.ClinicalDetails.SymptomStartDate?.ToString("dd-MMM-yyyy") </div>
+        </div>
+        <div class="nhsuk-grid-column-one-third">
+            <div class="notification-details-label"> Days from onset to treatment </div>
+            <div class="empty-cell-min-height"> @((@Model.Notification.ClinicalDetails.TreatmentStartDate?.Date - @Model.Notification.ClinicalDetails.SymptomStartDate?.Date)?.Days) </div>

```

Comment:
Hmm, there's actually a bunch of these transformations around..
I wonder if we want to have a `DisplayNotification.cs` model, which would commonise all of these, as well as the ones performed in the controller atm - then it would be the only BoundProprerty on the page?

Reply:
That ties in with what I said above :) I definitely think some refactoring might be in order! There is the formatted date model too, which is reused in multiple places so probably should be in its own model